src/plugins/Prism.ts

```diff
@@ -21,6 +21,7 @@ export const LANGUAGES = {
   ruby: "Ruby",
   sql: "SQL",
   typescript: "TypeScript",
+  yaml: "Yaml",
```

Comment:
```suggestion
  yaml: "YAML",
```

YAML is an acronym: https://en.wikipedia.org/wiki/YAML

Reply:
If you fix this up we can land the addition

Reply:
@tommoor change has been applied